MANUAL.adoc

```diff
@@ -274,6 +274,79 @@ public class SimpleInterfaceExample {
 [id=creating-fields]
 == Creating fields
 
+Fields may be added to types using the `field`, `staticField`, and `constantField` methods.
+These methods accept a consumer for a `StaticFieldCreator` or `InstanceFieldCreator` instance.
+The `field` method returns a <<fielddesc,`FieldDesc`>> for the newly created field.
+The `staticField` and `constantField` methods return a <<fields,`StaticFieldVar`>> which may be used directly in expressions.
+
+Fields may be initialized directly with a constant value at the time the field is defined by using the `setInitial` method.
+For more complex initializers, the `setInitializer` method may be used, which accepts a <<blocks,code block creator>> which must yield a value for the field.
+Initializers are automatically emitted into constructors or static initializers in the order that they are established.
+
```

Comment:
Since the "Creating types" chapter says

> By convention, the parameter name `cc` is typically used for the `ClassCreator` or `InterfaceCreator` instance.

maybe we should say the similar thing here for field creators?

> By convention, the parameter name `fc` is typically used for the `FieldCreator` instance.